frontend/src/components/PercentageBar.module.scss

```diff
@@ -0,0 +1,16 @@
+.stakes {
+  display: flex;
+  border-radius: 10px;
+  overflow: hidden;
+
+  // horizontal
+  width: 100%;
+  height: max-content;
+  flex-direction: row;
+
+  &.vertical {
+    height: 100%;
+    width: max-content;
+    flex-direction: column;
+  }
```

Comment:
I'd change the `horizontal` to it's own class just like the vertical one for consistency. Or if you want to keep horizontal as the default you could just change it to:

```
@import '../shared.scss';

.stakes {
  @extend .layout-row;
  border-radius: 10px;
  overflow: hidden;
  width: 100%;
  height: max-content;

  &.vertical {
    height: 100%;
    width: max-content;
    flex-direction: column;
  }
}
```